src/components/policy/policy_external/include/policy/policy_types.h

```diff
@@ -276,6 +276,7 @@ struct AppPermissions {
       : application_id(app_id)
       , isAppPermissionsRevoked(false)
       , appRevoked(false)
+      , isSDLAllowed(false)
       , appPermissionsConsentNeeded(false)
       , appUnauthorized(false)
```

Comment:
```suggestion
      , appPermissionsConsentNeeded(false)
      , appUnauthorized(false)
      , isSDLAllowed(false)
```
The `isSDLAllowed` initialization should be moved to match the declaration order